phylogenetic/rules/export.smk

```diff
@@ -139,6 +139,7 @@ rule export:
         clades = lambda wildcard: "results/{gene}/clades_{serotype}.json" if wildcard.gene in ['genome'] else [],
         nt_muts = "results/{gene}/nt-muts_{serotype}.json",
         aa_muts = "results/{gene}/aa-muts_{serotype}.json",
+        description = config["export"]["description"],
```

Comment:
This needs to be passed to the `augur export` command below with `--description {input.description}`.

Reply:
Gah, I should have caught that, thanks! Fixed in https://github.com/nextstrain/dengue/pull/71/commits/b9013020c768ccc3eab1120c9ae3adb0686c7413